src/include/miopen/find_solution.hpp

```diff
@@ -154,8 +154,8 @@ auto FindSolutionImpl(rank<0>,
                       Db&&,
                       const AnyInvokeParams&,
                       const std::string&,
-                      const std::optional<FindOptions>&)
-    -> decltype(s.GetSolution(context, problem))
+                      const std::optional<FindOptions>&) -> decltype(s.GetSolution(context,
```

Comment:
Changes in this file only contain formatting changes, which are not relevant to the core functionality. I think we should exclude these purely stylistic changes to keep the PR focused..?

Reply:
resolved